warplib/src/warp_memory.cairo

```diff
@@ -1,6 +1,9 @@
 use dict::Felt252Dict;
 use dict::Felt252DictTrait;
-
+use array::Array;
```

Comment:
Yea, I'd like the changes to be consistent with each other. I'd be happy to hear @rodrigo-pino opion on write memory traits and read memory traits since they seem contradictory to the approach I propose in the PR. Do we plan to operate on felts directly? My idea would be to separate existing memory trait into low level access trait operating on felts and an external interface which is high level operating on actual types. In my view only the external interface should be used directly (kind of public-private distinction)